src/components/Editor.svelte

```diff
@@ -56,7 +56,10 @@
       automaticLayout: true,
       readOnly: true,
       folding: true,
-      quickSuggestions: false
+      quickSuggestions: false,
+      wordWrap: "on",
+      unicodeHighlight: {invisibleCharacters: false,
```

Comment:
Why shouldn't invisible characters be highlighted?

Reply:
Fair question, I'm not really sure about this one. I use Birman keyboard layout, so there are things like non-breaking spaces that I don't want to be that much noticable. :) On the other side, it could be useful for  to see that unusual behaviour to understand the layout (or prevent some hidden data when copy-pasting). Personally I would like to be able to disable it, though.

I have not seen any of that in WebApp, so I thought that we could disable that feature for now, but after I thought a bit I agree that highlighting invisible should be useful.